### ISSUE TITLE
lr_scheduler.step() before optimizer.step() Error while executing the code

### ISSUE THREAD

**Issue by anshudaur** [microsoft/IRNet#45]
HI, 

I am getting error while executing irnet on colab. I was able to run IRNET+bert on colab but not with glove. I also tried downgrading to the older version of pytorch, but still the execution of train.py stops without any warning

\lr_scheduler.py:123: UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
  "https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate", UserWarning)

Thanks for your help.
Anshu
